bin/metaBackend.py

```diff
@@ -21,47 +21,506 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 
 """
-This is a program for ingesting information into LSST Metadata Server.
+Metadata Server client program. It is currently used to ingest information into the
```

Comment:
This file combines Python client API and command line parser, I think we should separate them (API should be in separate python module).